blocks/library/cover-image/index.js

```diff
@@ -177,3 +183,12 @@ registerBlockType( 'core/cover-image', {
 		);
 	},
 } );
+
+function dimRatioToClass( ratio ) {
+	let rounded = Math.round( ratio / 10 );
+	// String#padStart not ready for prime time
```

Comment:
This a difficult task to implement correctly, and I think the only viable option is to call out to a service which is designed to take on the burden of maintaining this implementation: http://left-pad.io/ :trollface: 

On a more serious note, there's: https://lodash.com/docs/4.17.4#padStart

Or my personal preference, the negative slice:

```js
( '0' + rounded ).slice( -2 );
```

Reply:
> there's lodash

Ooh, Dash must have failed me. :) Though I like the negative slice better.